### DISCUSSION TITLE
Feature request: CTRL+Click on a test in the test pane should open the test source

### DISCUSSION THREAD

**Discussion by stevebeauge** [orta/vscode-jest#987] [Ideas]
Currently, I can right click on a test and click on choose "Go to test" to navigate to the test source.

I think it may be cool to be able to Ctrl+Click or double click or whatever to do the same.

Thanks

**Comment by connectdotz** [comment]
This feature is provided by vscode, you can file a request there to ask them to publish this command so you can associate hot-key with it.
